src/options/output.rs

```diff
@@ -275,6 +275,6 @@ pub enum JsonVersion {
     #[serde(rename = "1")]
     #[default]
     One,
-    #[serde(rename = "2")]
-    Two,
```

Comment:
If you leave this value here does it create a compiler warning? It feels a little odd if somebody was to just look at this file and see it go from one to two.one

Reply:
Yeah the compiler complains